src/rules/noImplicitDependenciesRule.ts

```diff
@@ -89,6 +93,61 @@ function walk(ctx: Lint.WalkContext<Options>) {
         }
         return dependencies.has(module);
     }
+
+    function findSimilarPackage(module: string): string | undefined {
```

Comment:
is there perhaps a (supported, widely used, well-tested) npm package we can use that does this rather than inlining a rather complex algorithm?

Reply:
@giladgray Hi! Thank you for your comment. I thought about using some external package but I didn't know if we want to put another third-party into the Tslint's dependencies. That said, IMO there's only one package that's looks reliable (https://github.com/aceakash/string-similarity) but requires Lodash. Please check it out and tell me what you think.